### ISSUE TITLE
Target is not instantiable.

### ISSUE THREAD

**Issue by asivaneswaran** [maikealame/laravel-auto#1]
Hi,

I followed the instructions but I get this error: 

`Target [Auto\Contracts\AutoInterface] is not instantiable.`

Any ideas why?

**Comment by maikealame** [comment]
Hey,
Can you provide some code and laravel version?

**Comment by asivaneswaran** [comment]
I added the Service Provider and this is my models file:

```
class Season extends Model
{
    use SoftDeletes, AutoWhere, AutoSort, AutoPaginate;

    protected $fillable = ['name', 'slug', 'start', 'end', 'registration_end', 'deadline', 'abbr'];

    protected $dates = ['start', 'end', 'registration_end', 'deadline'];
```


My controller:

```
$list = Season::autoWhere()->autoSort()->autoPaginate(10);
```

And this is the error:


```
BindingResolutionException in Container.php line 887:
Target [Auto\Contracts\AutoInterface] is not instantiable.
```


Thanks,

**Comment by maikealame** [comment]
Try this command:
`php artisan clear-compiled`
if you get no seccess, send me your laravel version and I will install from beginning to find this error.

**Comment by asivaneswaran** [comment]
Hey,

I am having the same issue.

I have a fresh Laravel project with version 5.4.23

Thanks

**Comment by maikealame** [comment]
Please, update this dependencie and try again. I fixed in 0.13.14 version

**Comment by asivaneswaran** [comment]
Ok it works now!

Thanks,
